requirements-dev.txt

```
@@ -1,5 +1,5 @@
 chardet==3.0.4
-nose==1.3.7
+pytest==6.2.5
 tox==3.20.1
 Cython==0.29.21
```

```suggestion
Cython==0.29.30
```
For Python 3.11-beta.4